.changeset/lazy-buckets-peel.md

```diff
@@ -0,0 +1,11 @@
+---
+"@neo4j/cypher-builder": patch
+---
+
+Deprecate apoc functions and procedures. These will no longer be supported in version 3 of Cypher Builder:
```

Comment:
I think it could be helpful to provide reasoning for this deprecation, as it has no alternatives in the message. Maybe it's worth redirecting users to users to use `Cypher.Function`.